config/packages/twig_extensions.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+    _defaults:
+        public: false
+        autowire: true
+        autoconfigure: true
+
+    # Uncomment any lines below to activate that Twig extension
```

Comment:
Shouldn't we remove this?

Reply:
It will become an application config, so in my opinion, it should contain explanatory comments and commented sample config.